hackathon_site/dashboard/frontend/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -11,20 +11,25 @@ import {
 } from "components/dashboard/ItemTable/ItemTable";
 import ProductOverview from "components/inventory/ProductOverview/ProductOverview";
 import Header from "components/general/Header/Header";
-import {
-    cardItems,
-    members,
-    teamCode,
-    mockPendingOrders,
-    mockCheckedOutOrders,
-} from "testing/mockData";
+import { cardItems, mockPendingOrders, mockCheckedOutOrders } from "testing/mockData";
 import { hackathonName } from "constants.js";
-import { useDispatch } from "react-redux";
+import { useDispatch, useSelector } from "react-redux";
 import { getHardwareWithFilters, setFilters } from "slices/hardware/hardwareSlice";
 import { getCategories } from "slices/hardware/categorySlice";
+import {
+    getCurrentTeam,
+    isLoadingSelector,
+    teamSelector,
+} from "slices/event/teamSlice";
+import LinearProgress from "@material-ui/core/LinearProgress";
 
 const Dashboard = () => {
     const dispatch = useDispatch();
+    const currentTeam = useSelector(teamSelector);
+    const memberNames = currentTeam?.profiles.flatMap(
```

Comment:
let's take advantage of redux and make a selector for the team code and member list

Reply:
sounds good!